cellxgene_schema_cli/cellxgene_schema/atac_seq.py

```diff
@@ -323,7 +323,9 @@ def validate_fragment_stop_coordinate_within_chromosome(parquet_file: str, annda
 
 def validate_fragment_read_support(parquet_file: str) -> Optional[str]:
     # check that the read support is greater than 0
-    df = ddf.read_parquet(parquet_file, columns=["read support"], filters=[("read support", "<=", 0)])
+    df = ddf.read_parquet(
+        parquet_file, columns=["read support"], filters=[("read support", "<=", 0)], keep_default_na=False
+    )
```

Comment:
I don't see `keep_default_na` as an arg for read_parquet, though I do see it passes through kwargs. Just wanted to check if this is doing what is intended / curious what its meant to be passed through to @Bento007

Reply:
it should be removed